scripts/re_extract.py

```
@@ -0,0 +1,181 @@
+    if source_filter:
+        print(f"             소스 필터: {source_filter.upper()}")
+    print("-" * 60)
+
+    updated = 0
+    skipped = 0
+    errors = 0
+
+    for i, path in enumerate(files, 1):
+        try:
+            data = json.loads(path.read_text(encoding="utf-8"))
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)